debian/control

```diff
@@ -10,8 +10,10 @@ Homepage: https://learningequality.org/kolibri
 
 Package: kolibri-server
 Architecture: all
-Depends: kolibri (>= 0.12.6), nginx-full, uwsgi (>= 2.0.12), uwsgi-plugin-python3, redis-server (>=4.0)
 Recommends: anacron
+Depends: kolibri (>= 0.12.6), nginx-full, uwsgi (>= 2.0.12), uwsgi-plugin-python3, redis-server (>=4.0), python3 (>= 3.4)
```

Comment:
Why adding python 3.4 dependency?

Reply:
There is a linting error that we have Python scripts (`kolibri_server_setup.py`) without a Python dependency.

Reply:
yeah, my question was more why `>=3.4` : I don't think there's anything in that script that would not work on python 3.0